In [1]:
import pandas as pd
from scipy import stats

In [4]:
df=pd.read_csv("/content/BuyerRatio.csv")

In [5]:
df.head()

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

In [8]:
stats.chi2_contingency([df["East"],df["West"],df["North"],df["South"]])

(1.5959455386610577,
 0.6603094907091882,
 3,
 array([[  42.76531299,  442.23468701],
        [ 146.81287862, 1518.18712138],
        [ 131.11756787, 1355.88243213],
        [  72.30424052,  747.69575948]]))

P value= 0.66 which is greater than alpha value so H0 is accepted